website/src/controllers/api-v3/quests.js

```diff
@@ -65,4 +65,58 @@ api.inviteToQuest = {
   },
 };
 
+/**
+ * @api {post} /groups/:groupId/quests/reject Reject a quest
+ * @apiVersion 3.0.0
+ * @apiName RejectQuest
+ * @apiGroup Group
+ *
+ * @apiParam {string} groupId The group _id (or 'party')
+ * @apiParam {string} questKey The quest _id
+ *
+ * @apiSuccess {Object} Quest Object
+ */
+api.rejectQuest = {
+  method: 'POST',
+  url: '/groups/:groupId/quests/reject',
+  middlewares: [authWithHeaders(), cron],
+  async handler (req, res) {
+    let user = res.locals.user;
+    let questKey = user.party.quest.key;
+    let quest = questScrolls[questKey];
+
+    req.checkParams('groupId', res.t('groupIdRequired')).notEmpty();
+
+    let validationErrors = req.validationErrors();
+    if (validationErrors) throw validationErrors;
+
+    let group = await Group.getGroup({user, groupId: req.params.groupId, fields: 'type quest'});
+    if (!group) throw new NotFound(res.t('groupNotFound'));
+    if (!quest) throw new NotFound(res.t('questNotFound', { key: questKey }));
```

Comment:
Sure. I was thinking there was some added security with this, but I don't really see that being the case. I think it is good to remove. Sound good to you?